gensim/models/ldaseqmodel.py

```diff
@@ -172,6 +167,8 @@ def __init__(self, corpus=None, time_slice=None, id2word=None, alphas=0.01, num_
 
         # if a corpus and time_slice is provided, depending on the user choice of initializing LDA, we start DTM.
         if corpus is not None and time_slice is not None:
+            self.max_doc_len = max(len(line) for line in corpus)
```

Comment:
move down because of this and original code is not supported case `corpus=None `.